packages/klighd-core/README.md

```diff
@@ -51,16 +61,19 @@ import {
     SetPreferencesAction,
     bindServices,
 } from "@kieler/klighd-core";
-// Your implementation of the `Connection` interface.
+// Your implementation of the `Connection` interface
 import { ConnectionImpl } from "./services/connection";
+// Your implementation of the `PersistenceStorage` interface
```

Comment:
I think this should link to the default implementation, if nothing should be changed for the specific extension.

Reply:
What default implementation? There exists no default implementation in `klighd-core`. This is supposed to provide an idea about how an integration of `klighd-core` can be approached.

Reply:
I thought this was the readme for the extension, where a default implementation should be available. Did not read the file path correctly during review.